spec/pagseguro/pre_approval_request/response_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "spec_helper"
+
+describe PagSeguro::PreApprovalRequest::Response do
+  context "when pre approval request is created" do
+    def xml_response(path)
+      response = double(
+        body: File.read("./spec/fixtures/#{path}"),
+        code: 200,
+        content_type: "text/xml",
+        "[]" => nil
+      )
+
+      Aitch::Response.new({xml_parser: Aitch::XMLParser}, response)
```

Comment:
Maybe use `Fakeweb` here? Just saying 'cause I think it's good to pick a tool and only use it :grin: 

Here's an example https://github.com/pagseguro/ruby/pull/94/files#diff-64ab9060d41e26121cba913b9d93701aR8 :smile: